website/templates/emails/contributor_added_draft_registration.html.mako

```diff
@@ -8,19 +8,33 @@
     %>
     Hello ${user.fullname},<br>
     <br>
-    ${referrer_name + ' has added you' if referrer_name else 'You have been added'} as a contributor to the draft registration "${node.title}" on OSF: ${node.absolute_url}<br>
-    <br>
-    You will ${'not receive ' if all_global_subscriptions_none else 'be automatically subscribed to '}notification emails for this project. To change your email notification preferences, visit your project or your user settings: ${settings.DOMAIN + "settings/notifications/"}<br>
-    <br>
-    If you are erroneously being associated with "${node.title}," then you may visit the draft registration and remove yourself as a contributor.<br>
-    <br>
-    Sincerely,<br>
-    <br>
-    OSF Robot<br>
-    <br>
-    Want more information? Visit https://osf.io/ to learn about the Open Science Framework, or https://cos.io/ for information about its supporting organization, the Center for Open Science.<br>
-    <br>
-    Questions? Email ${osf_contact_email}<br>
 
+    ${'You just started' if not referrer_name else referrer_name + ' just added you to'}  a new registration draft titled <a href="${node.absolute_url}">${node.title if node.title else 'new registration draft'}</a>
```

Comment:
duplicating the words "new registration draft" in the no-title case. Should omit this piece and use first instance of "new registration draft" as the link in the no-title case